src/main/java/org/openstreetmap/atlas/geography/atlas/command/PBFToAtlasSubCommand.java

```diff
@@ -0,0 +1,214 @@
+package org.openstreetmap.atlas.geography.atlas.command;
```

Comment:
PBF -> OsmPbf to stick with the already used convention with OsmPbf classes in the pbf command